app-next/components/MealByID/MealsDetails.jsx

```diff
@@ -0,0 +1,86 @@
+"use client";
+
+import MealReviews from "./MealReviews";
+
+export default function MealDetails({ meal, onReserveClick }) {
+  if (!meal) return null;
+
+  return (
+    <div className="relative bg-white shadow-xl rounded-2xl border-2 border-orange-600 max-w-6xl mx-auto mt-24 px-6 md:px-10 py-5">
+      {/* Top Center circle image */}
```

Comment:
When you feel a need for a comment -- this is a good sign that it can be a separate component